src/pip/_internal/utils/appdirs.py

```diff
@@ -13,9 +13,7 @@
 from pip._internal.utils.typing import MYPY_CHECK_RUNNING
 
 if MYPY_CHECK_RUNNING:
-    from typing import (  # noqa: F401
-        List, Union
-    )
+    from typing import List
```

Comment:
I'd be fine with keeping Union here in this commit, and removing it the next commit.

Reply:
There's a few places other places too which remove not-used imports, it'd be nice if all of those changes could be moved into a separate commit.